sentry-kubernetes.py

```diff
@@ -116,16 +117,19 @@ def watch_loop():
         elif 'namespace' in meta:
             namespace = meta['namespace']
 
+        if event.involved_object and event.involved_object.kind:
+            kind = event.involved_object.kind
+
         if event.involved_object and event.involved_object.name:
             name = event.involved_object.name
-            bits = name.split('-')
-            if len(bits) in (1, 2):
-                short_name = bits[0]
+            if kind in MANGLE_NAMES:
```

Comment:
I think we'd want this to be `if not MANGLE_NAMES or kind in MANGLE_NAMES:` (in other words: default to the existing style) because otherwise all existing users will have to set the names they want to mangle. This would make it an opt-in thing, does that make sense or am I reading it wrong?

Reply:
Ahh, you are exactly right. Sorry about that.